web/modules/custom/pipeline/src/PipelineOrchestratorInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\pipeline;
+
+/**
+ * Provides an interface for the pipeline orchestrator.
+ */
+interface PipelineOrchestratorInterface {
+
+  /**
+   * Executes the orchestrator.
```

Comment:
This is very vague and I have no clue if I am right but it seems to me that an orchestrator (as mentioned in the orchestrator class) is merely a tracker for the step and the state while a pipeline is a series of steps, so in this term, the pipeline is the one executed, not the orchestrator.